### ISSUE TITLE
Add Construction Data

### ISSUE THREAD

**Issue by AllieDeford** [Hackcessible/hackcessible-webapp#9]

**Comment by Reagankm** [comment]
[SDOT Street Use Permit data](https://data.seattle.gov/Transportation/SDOT-Street-Use-Permit-by-Street-Segment-includes-/w47m-dg37) includes fields marking whether a sidewalk or street is closed and the start and end dates of the closure and even what particular times of day they are closed.  The permits seem to include addresses and text descriptions of the closure, though, nothing so helpful as lat/lon.  I was thinking there might be a way to look up a lat/lon based on an address, though, so we can put a pin to mark construction and people can click on it to see the description, even if we can't have a line that covers the entire closed sidewalk.

**Comment by AllieDeford** [comment]
There is a way to look up lat/lon based on address! It's called Geocoding. There is a handy Python library called [geopy](https://github.com/geopy/geopy) that can help you either use the Google Geocoder API or another Geocoder API (like the OpenStreetMaps one)

**Comment by Reagankm** [comment]
I used the data source for Street Use Permits by Address (https://data.seattle.gov/Transportation/SDOT-Street-Use-Permit-by-Address-includes-Impacts/czy3-hkh9) and just pulled a JSON file that was already filtered for permits open on 4/29.  The dataset includes latitude and longitude already so no geocoding was necessary.  Way too many invalid hits though even just looking at permits that said they were closing the sidewalk.  To narrow down scope I filtered to only show markers for permits that included the word "sidewalk" in their description and did not include "no mobility impact".  There are going to be some false positives and missed positives, I'm afraid.  We'll need a better way to refine this data.  There does not seem to be any standardization in permit descriptions and some things are marked "Sidewalk closed" when it's clear from the description that really there's just a food truck at that spot (impacting but not closing the sidewalk).
